Leyrin-Aguilar/tarea-3/Duplicate sandwich/script.js

```diff
@@ -0,0 +1,8 @@
+function duplicateSandwich(a) {
+  for (let i = 0; i < a.length - 1; i++)
```

Comment:
@LeyrinAguilar  , use modern approaches for array operations like forEach, map, reduce, in fact this one could be a one line solution